docs/cookbook/adding-a-new-advert-position.md

```diff
@@ -0,0 +1,51 @@
+# Adding a New Advert Position
+
+By default, you can place advert banner on predefined positions:
+- under the page header
+- in the left panel (under the category tree)
+- above the page footer
+- in a category detail (above the category name)
+
+All the positions are defined in `AdvertPositionRegistry` class 
+that provides them for displaying on adverts list in admin
```

Comment:
Having every sentence on a single line helps with the diffs and further maintenance. Consider removing the line breaks.